papis/docmatcher.py

```diff
@@ -28,8 +28,21 @@ class DocMatcher(object):
     """
     search = ""  # type: str
     parsed_search = []  # type: List[Any]
-    doc_format = '{%s[DOC_KEY]}' % (papis.config.getstring('format-doc-name'))
     logger = logging.getLogger('DocMatcher')
+
+    formater = papis.config.getstring("formater")
+    if formater == "python":
+        doc_format = '{%s[DOC_KEY]}' % (
+            papis.config.getstring('format-doc-name'))
+
+    elif formater == "jinja2":
+        doc_format = '{{%s["DOC_KEY"]}}' % (
+            papis.config.getstring('format-doc-name'))
```

Comment:
Could this use `papis.format.format`? Ideally, we don't want to hardcode the available formatters and just use the plugin system somehow.

Reply:
Indeed, it is probably a good idea to implement the `doc_format` variable directly in the `papis.format.Formater` themselves, rather than hard-coding it in the `DocMatcher`.